app/components/listing/ServiceDetails.tsx

```diff
@@ -1,209 +0,0 @@
-import React, { useState } from "react";
-import ReactMarkdown from "react-markdown";
-import { Link } from "react-router-dom";
-import { removeAsterisksAndHashes } from "utils/strings";
-import { NotesList } from "./NotesList";
-import { Service, RecurringSchedule } from "../../models";
-
-import ServiceVerified from "../../assets/img/ic-attributed-record.svg";
-import "./ServiceDetails.scss";
-
-// Where is this rendered?
-
-export const ServiceDetails = ({ service }: { service: Service }) => {
```

Comment:
@adriencyberspace you also questioned this was used anywhere. I grep'd the code, it's nowhere to be found. ✂️